config/jobs/kubernetes-sigs/azuredisk-csi-driver/azuredisk-csi-driver-config.yaml

```diff
@@ -214,6 +214,11 @@ presubmits:
         base_ref: release-1.7
         path_alias: sigs.k8s.io/cluster-api-provider-azure
         workdir: true
+      - org: kubernetes-sigs
+        repo: cloud-provider-azure
+        base_ref: master
```

Comment:
should this be release-1.26?

Reply:
No, KUBERNETES_VERSION is set to `latest` (i.e. master branch) which means we should test with latest CCM according to this version matrix: https://github.com/kubernetes-sigs/cloud-provider-azure#version-matrix

Reply:
ditto my below comment